scripts/deploy_bug.py

```diff
@@ -15,8 +15,6 @@
 Then copy and paste the bits it prints out.
 """
 
-from __future__ import print_function
```

Comment:
This is a thing we needed in Python 2, but in Python 3, print *is* a function so we don't need these anymore.